sql/core/src/test/scala/org/apache/spark/sql/CollationSuite.scala

```diff
@@ -1625,6 +1626,63 @@ class CollationSuite extends DatasourceV2SQLBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("Expressions not supporting CS_AI collators") {
```

Comment:
this shouldn't be in `CollationSuite`, please see `CollationStringExpressionsSuite`

Reply:
we use `CollationStringExpressionsSuite` for expression-level unit tests